packages/web3-core/src/index.ts

```diff
@@ -15,17 +15,17 @@ You should have received a copy of the GNU Lesser General Public License
 along with web3.js.  If not, see <http://www.gnu.org/licenses/>.
 */
 
-export * from './web3_config';
-export * from './web3_request_manager';
-export * from './web3_subscription_manager';
-export * from './web3_subscriptions';
-export * from './web3_context';
-export * from './web3_batch_request';
-export * from './utils';
-export * from './types';
-export * from './formatters';
-export * from './web3_promi_event';
-export * from './web3_event_emitter';
+export * from './web3_config.js';
```

Comment:
Thanks @luu-alex , could you run blackbox tests on this branch ( locally or in github ) for looking balcbox is also passing.

Reply:
👍  https://github.com/web3/web3.js/actions/runs/5134874349/jobs/9239775016 all other ci tests were run in a previous commit and it passed